Binance.Net/Objects/BinanceTransfer.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Binance.Net.Converters;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using CryptoExchange.Net.Converters;
+
+namespace Binance.Net.Objects
+{
+    /// <summary>
+    /// The result of transfering from main to margin account or margin to main
+    /// </summary>
+    public class BinanceTransfer
```

Comment:
`BinanceTransfer`, `BinanceRepay` and `BinanceBorrow` are all the same, could be a single class

Reply:
Sure .good idea.I will call like `BinanceTransaction`

Reply:
How it must call?

Reply:
Maybe add margin in the name, what about `BinanceMarginTransaction`?